sql/telemetry_derived/devtools_accessiblility_panel_usage_v1/query.sql

```diff
@@ -0,0 +1,134 @@
+      "KEYBOARD"
+    ) AS audit_keyboard_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+      "TEXT_LABEL"
+    ) AS audit_text_label_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_accessible_context_menu_item_activated,
+      "print-to-json"
+    ) AS print_to_json_context_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.deuteranomaly"
+    ) AS simulation_deuteranomaly_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.protanomaly"
+    ) AS simulation_protanomaly_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.protanopia"
+    ) AS simulation_protanopia_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.deuteranopia"
+    ) AS simulation_deuteranopia_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.tritanopia"
+    ) AS simulation_tritanopia_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.tritanomaly"
+    ) AS simulation_tritanomaly_count,
+    udf.get_key(
+      payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+      "accessibility.simulation.contrastLoss"
+    ) AS simulation_contrast_loss_count
+  FROM
+    telemetry.main
+  WHERE
+    sample_id = 1
+    AND DATE(submission_timestamp) = @submission_date
+    AND payload.processes.parent.scalars.devtools_accessibility_opened_count > 0
+)
+SELECT
+  submission_date,
+  normalized_channel,
+  COUNT(*) AS clients,
+  SUM(opened_count) AS panel_opens,
+  SUM(context_menu_opened_count) / (COUNT(*)) AS context_menu_opened_per_client_day,
+  SUM(print_to_json_context_count) / (COUNT(*)) AS print_to_json_context_per_client_day,
+  COALESCE(SUM(CAST(print_to_json_context_count > 0 AS INT64)), 0) AS print_to_json_context_users,
+  SUM(node_inspected_count) / (COUNT(*)) AS node_inspected_per_client_day,
+  COALESCE(SUM(CAST(node_inspected_count > 0 AS INT64)), 0) AS node_inspected_users,
+  COALESCE(SUM(CAST(picker_used_count > 0 AS INT64)), 0) AS picker_users,
+  COALESCE(SUM(CAST(picker_used_count > 0 AS INT64)), 0) / (
+    COUNT(*)
+  ) AS picker_users_per_client_day,
+  SUM(picker_used_count) / (COUNT(*)) AS picker_count_per_client_day,
+  SUM(browser_context_count) / (COUNT(*)) AS browser_context_per_client_day,
+  COALESCE(SUM(CAST(browser_context_count > 0 AS INT64)), 0) AS browser_context_users,
```

Comment:
Looking more carefully at the logic, I think the best way to get to correct counts is to make the initial query aggregate per client, and then these metrics would become:

```suggestion
  SAFE_DIVIDE(SUM(browser_context_count), (COUNT(*)) AS browser_context_per_client_day,
  COUNTIF(browser_context_count > 0) AS browser_context_users,
```